mips_api/mips.py

```diff
@@ -9,33 +10,50 @@ class App:
     _mips_url_chart = 'https://mipapi.abilaonline.com/api/v1/maintain/chartofaccounts'
     _mips_url_logout = 'https://mipapi.abilaonline.com/api/security/logout'
 
+    # add these meta codes as valid `CostCenter` values
+    _extra_program_codes = {
+        '000000':  'No Program',
+        '000001':  'Other',
```

Comment:
i thought we agreed that this data should just be added to mips?

Reply:
yeah but I'm still waiting to hear back about whether or not that would be disruptive to finance